includes/Modules/Analytics_4.php

```diff
@@ -926,7 +922,8 @@ function ( $dimension_def ) {
 				} catch ( Invalid_Report_Dimensions_Exception $exception ) {
 					return new WP_Error(
 						'invalid_analytics_4_report_dimensions',
-						$exception->getMessage()
+						$exception->getMessage(),
+						array( 'status' => 400 )
```

Comment:
I can see why you used the status here as well but let's not do that because it would imply we should make the same changes in the other modules that also implement this check. Aside from that, I think the default status is okay here because the client is not making an invalid request – the validation is applied due to internal state/conditions related to dashboard sharing.

Reply:
No worries and thanks for changing it back :+1: